package.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
We should be sure to add `node_modules` to `.gitignore`.

I guess we should add `package-lock.json` to the repository?

Reply:
Made both of these changes